src/modules/fancyzones/lib/FancyZonesData.h

```diff
@@ -0,0 +1,145 @@
+#pragma once
+
+#include "JsonHelpers.h"
+
+#include <common/settings_helpers.h>
+#include <common/json.h>
+#include <mutex>
+
+#include <string>
+#include <unordered_map>
+#include <optional>
+#include <vector>
+#include <winnt.h>
+
+namespace FancyZonesDataTypes
+{
+    struct ZoneSetData;
+    struct DeviceInfoData;
+    struct CustomZoneSetData;
+    struct AppZoneHistoryData;
+}
+
+#if defined(UNIT_TESTS)
+namespace FancyZonesUnitTests
+{
+    class FancyZonesDataUnitTests;
+    class FancyZonesIFancyZonesCallbackUnitTests;
+    class ZoneSetCalculateZonesUnitTests;
+    class ZoneWindowUnitTests;
+}
+#endif
+namespace FancyZonesDataNS
+{
+    const std::wstring& GetActiveZoneSetTmpPath();
+    const std::wstring& GetAppliedZoneSetTmpPath();
+    const std::wstring& GetDeletedCustomZoneSetsTmpPath();
+
+    class FancyZonesData
+    {
+    public:
+        FancyZonesData();
+
+        inline const std::wstring& GetPersistFancyZonesJSONPath() const
```

Comment:
Same thing with `GetPersistFancyZonesJSONPath` and `GetPersistAppZoneHistoryFilePath`

Reply:
Same as above, deleted :)